docker-compose.yml

```diff
@@ -40,7 +40,7 @@ services:
     ports:
       - '8000:8000'
     environment:
-      FRONTEND.ADDRESS: http://frontend:3000
+      FRONTEND.ADDRESS: http://localhost:3000
```

Comment:
so what's the point of this change? For me it breaks CORS, so... reason?
![image](https://github.com/Veermove/zpi/assets/75857174/0dca2977-1d60-482c-be21-e1131019d087)

Reply:
![image](https://github.com/Veermove/zpi/assets/75857174/cf48a89d-9462-43bb-93b7-b02da3fec70d)
No problem. Please keep in mind that note that @gcfbn added to README.md (last line)